README.md

```diff
@@ -0,0 +1,155 @@
+
+## Instructions
+
+### File Structure
+
+```
+├── CONTRIBUTING.md
+├── Gemfile
+├── Gemfile.lock
+├── LICENSE.md
+├── README.md
+├── Rakefile
+├── app
+│   ├── controllers
+│   │   └── application_controller.rb
+│   ├── models
+│   │   ├── tweet.rb
+│   │   └── user.rb
+│   └── views
+│       ├── index.erb
+│       ├── layout.erb
+│       ├── tweets
+│       │   ├── create_tweet.erb
+│       │   ├── edit_tweet.erb
+│       │   ├── show_tweet.erb
+│       │   └── tweets.erb
+│       └── users
+│           ├── create_user.erb
+│           └── login.erb
+├── config
+│   └── environment.rb
+├── config.ru
+├── db
+│   ├── development.sqlite
+│   ├── migrate
+│   │   ├── 20151124191332_create_users.rb
+│   │   └── 20151124191334_create_tweets.rb
+│   ├── schema.rb
+│   └── test.sqlite
+└── spec
+    ├── controllers
+    │   └── application_controller_spec.rb
+    └── spec_helper.rb
+```
+
+### Gemfile and environment.rb
+
+This project is supported by Bundler and includes a `Gemfile`.
+
+Run bundle install before getting started on the project.
+
+As this project has quite a few files, an `environment.rb` is included that loads all the code in your project along with Bundler. You do not ever need to edit this file. When you see require_relative `../config/environment`, that is how your environment and code are loaded.
+
+### Models
+
+You'll need to create two models in `app/models`, one `User` model and one `Tweet`. Both classes should inherit from `ActiveRecord::Base`.
+
+### Migrations
+
+You'll need to create two migrations to create the users and the tweets table.
+
+Users should have a username, email, and password.
+
+Tweets should have content, and a user_id.
+
+### Associations
+
+You'll need to set up the relationship between users and tweets. Think about how the user interacts with the tweets, what belongs to who?
+
+
+### Home Page
+
+You'll need a controller action to load the home page. You'll want to create a view that will eventually link to both a login page and signup page.
+
+### Create Tweet
+
+You'll need to create two controller actions, one to load the create tweet form, and one to process the form submission. The tweet should be created and saved to the database.
+
+### Show Tweet
+
+You'll need to create a controller action that displays the information for a single tweet. 
+
+### Edit Tweet
+
+You'll need to create two controller actions to edit a tweet: one to load the form to edit, and one to actually update the tweet entry in the database.
+
+You'll want to create an edit link on the tweet show page.
+
+### Delete Tweet
+
+You'll need to two controller actions, one to display a form to delete a tweet, and one to process the form submission. 
+
+The delete form doesn't need to have any input fields, just a submit button.
+
+### Sessions
+
+In the controller, you'll want to enable sessions.
```

Comment:
oh just that one sentence of "you'll wnat to enable sessions". So it still should say line 138, but just don't remind them to enable sessions. I want them to remember that